processor/src/lib.rs

```diff
@@ -242,16 +245,22 @@ impl Process {
     fn execute_span_block(&mut self, block: &Span) -> Result<(), ExecutionError> {
         self.start_span_block(block)?;
 
+        let mut offset = 0;
+        let mut decorator_iter = block.decorator_iter();
```

Comment:
nit: let's rename the variables as follows:

* `offset` -> `op_offset`
* `decorator_iter` -> `decorators` (I don't think there is a need to specify the type via suffix).

Reply:
Done